esp-hal/Cargo.toml

```diff
@@ -99,7 +99,7 @@ serde        = { version = "1.0.219", default-features = false, features = ["der
 jiff = { version = "0.2.10", default-features = false, features = ["static"] }
 
 [features]
-default = ["rt"]
+default = ["rt", "esp32c6", "unstable"]
```

Comment:
I think you accidentally committed your patch to get rust analyzer working. I've done that a few times.